samples/client/others/rust/hyper/oneOf/src/apis/client.rs

```diff
@@ -6,6 +6,7 @@ use super::configuration::Configuration;
 pub struct APIClient {
```

Comment:
You might need to rebase / merge main - I wouldn't expect to see any changes for Rust

Reply:
The changes for Rust (among others) are caused by me adding new sample configs - the bugfixed scenario was previously not present in any of the examples.

Rebased onto upstream master, force pushed to remote as a part of good practice.

Reply:
Rebased again, resolved conflict.